examples/all-clusters-app/linux/fuzzing-main.cpp

```diff
@@ -73,7 +73,11 @@ extern "C" int LLVMFuzzerTestOneInput(const uint8_t * aData, size_t aSize)
     // For now, just dump the data as a UDP payload into the session manager.
     // But maybe we should try to separately extract a PeerAddress and data from
     // the incoming data?
-    Transport::PeerAddress peerAddr;
+
+    // dumping payload with random transport types
+    Transport::Type fuzzedTransportType = static_cast<Transport::Type>(*aData % 5);
```

Comment:
how about adding a kLast constant on the Transport::Type so that `5` is nto so magical?

Reply:
I added one and I name it Transport::Type::kNumofTypes since it seemed more descriptive to me, is that ok?